pybop/optimisers/pints_optimisers.py

```diff
@@ -194,12 +200,44 @@ def __init__(self, x0, sigma0=0.1, bounds=None):
         super().__init__(x0, sigma0, self.boundaries)
 
 
+class NelderMead(pints.NelderMead):
+    """
+    Implements the Nelder-Mead downhill simplex method from PINTS.
+
+    This is a deterministic local optimiser. In most update steps it performs
+    either 1 evaluation, or 2 sequential evaluations, so that it will not
```

Comment:
Just to make it clear it is not two options.

```suggestion
    either one evaluation, or two sequential evaluations, so that it will not
```